charts/docker-hub-rate-limit-exporter-chart/templates/servicemonitor.yaml

```diff
@@ -11,21 +11,40 @@ metadata:
     {{- if .Values.serviceMonitor.additionalLabels }}
     {{- toYaml .Values.serviceMonitor.additionalLabels | nindent 4 }}
     {{- end }}
-
 spec:
-  endpoints:
-    - port: http
-      path: /
-      {{- if .Values.serviceMonitor.interval }}
-      interval: {{ .Values.serviceMonitor.interval }}
-      {{- end }}
-      {{- if .Values.serviceMonitor.scrapeTimeout }}
-      scrapeTimeout: {{ .Values.serviceMonitor.scrapeTimeout }}
-      {{- end }}
-  namespaceSelector:
```

Comment:
You removed the namespaceSelector configuration. However, I think we need the configuration for the case that `.Values.serviceMonitor.namespace` is also set.

Depending on the prometheus configuration it is necessary to deploy the service monitor to a specific namespace. I that case, the ServiceMonitor Resource and the actual deployment / DeamonSet will be installed in separat namespaces and we need to set `namespaceSelector` to .Release.Namespace.

Reply:
yes mb i will put it back